tox.ini

```
@@ -27,24 +27,26 @@ commands =
 
 [testenv]
 extras = tests
-commands = python -m pytest {posargs}
+commands = python -m pytest {posargs:-n auto}
 
 
 [testenv:py36]
 extras = tests
 deps = coverage[toml]
-commands = coverage run -m pytest {posargs}
+setenv = COVERAGE_FILE = .coverage.{envname}
+commands = python -m pytest --cov --cov-append --dist load {posargs:-n auto}
```

Try explicitly pre-loading the pytest-cov plugin. This way, it's guaranteed to be loaded before all others. It should help defeat the most common cause of the `Coverage.py warning: No data was collected` warning.
```suggestion
commands = python -m pytest -p pytest_cov --cov --cov-append --dist load {posargs:-n auto}
```

---

nope: https://github.com/python-attrs/attrs/runs/7911673793?check_suite_focus=true :(